src/main/java/net/mcreator/element/converter/v2023_2/SpecialInformationConverter.java

```diff
@@ -67,7 +71,7 @@ public GeneratableElement convert(Workspace workspace, GeneratableElement input,
 				specialInformationField.set(object, new StringProcedure(null,
 						specialInfo.stream().map(info -> info.replace(",", "\\,")).collect(Collectors.joining(","))));
 			} catch (IllegalAccessException | NoSuchFieldException ignored) {
-
+				LOG.error(ignored.getMessage(), ignored);
```

Comment:
Usually we don't error such messages in the log, warn level should be fine.
Also `ignored` should instead be named e.g. `ex` as you're actually using it.